ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/user-provisioned-dns/provision/cucushift-installer-rehearse-aws-ipi-user-provisioned-dns-provision-chain.yaml

```diff
@@ -0,0 +1,13 @@
+chain:
+  as: cucushift-installer-rehearse-aws-ipi-user-provisioned-dns-provision
+  steps:
+  - chain: cucushift-installer-rehearse-aws-ipi-provision
+  - ref: cucushift-installer-check-user-provisioned-dns
+  env:
+  - name: USER_PROVISIONED_DNS
```

Comment:
When user dns enabled, can the cluster be reached outside without any custom DNS ? Because in the provision chain, I did not see any step to provision user custom dns for the cluster.

Reply:
Configuring custom DNS is a day-2 operation, once the basis function works, we will add that case in CI.